dev/content/Overlays.vue

```diff
@@ -128,18 +125,74 @@ const tooltipCopy = `<Tooltip>Here's something subtly helpful.</Tooltip>`
       <ComponentLayout class="mt-8" title="Flash">
         <template v-slot:description>
           We look for an array of flashes on <code>window.Flashes</code> and we
-          can emit events on the bus that render flashes.
+          can deploy flashes from anywhere with the useAppFlashes composable.
         </template>
 
         <div>
-          <label class="block text-sm font-medium text-gray-700">
-            <ClickToCopy :value="flashCopy" />
-          </label>
-          <div class="mt-1">
-            <button type="button" class="xy-btn" @click="flash()">
-              Show Me
-            </button>
-          </div>
+          <ul class="mt-1 space-y-2">
+            <li>
+              <InputLabel
+                label="Flash generic error using configured email address:"
+              />
+              <pre><code class="language-typescript">/* Import and do this anytime, best before createApp() or inside a root app level component */
```

Comment:
This UI is not breaking in its container but overflowing out the edges

Reply:
Ah, prism.js wasn't kicking in on the dynamic page change.  Made a quick update so that it works as expected, but will plan to revisit code samples in general.